templates/_base/base.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
```

Comment:
These "base" templates are used by `404.html` and `500.html` but not `index.html`? That seems odd...

Reply:
Oh again I guess this is because you copied these pages from jp.ubuntu.com.

So these includes should be replaced with the actual markup from `index.html`.

Reply:
Yeah this was copied from jp but the index is just the one copied from the GUI project with a few minor changes. I'll be updating the index file a lot in the coming PR's